app/sentry.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sentry from '@sentry/browser';
+import { Ember } from '@sentry/integrations';
+import config from 'open-event-frontend/config/environment';
+
+Sentry.init({
+    integrations: [new Ember()],
+    beforeSend(event: Sentry.Event) {
+        const exception = event.exception?.values?.[0];
+        const errorValue = exception?.value;
+        if (errorValue?.includes("Ember Data Request") && 
+            errorValue?.includes("404")) {
+            // Ignore 404 errors from Ember Data because 
+            // I don't know how to turn them off
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 spaces but found 12.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5010664)